Gluing/CartesianCategory.agda

```diff
@@ -0,0 +1,140 @@
+{-# OPTIONS --safe #-}
+{-# OPTIONS --lossy-unification #-}
+module Gluing.CartesianCategory where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Relation.Nullary hiding (⟪_⟫)
+open import Cubical.Data.Nat
+open import Cubical.Data.Nat.Properties
+open import Cubical.Data.Bool
+open import Cubical.Data.Sum as Sum
+
+open import Cubical.Categories.Category renaming (isIso to isIsoC)
+open import Cubical.Categories.Functor
+open import Cubical.Categories.Constructions.Free.CartesianCategory.Base as Law
```

Comment:
Law?

Reply:
Short for Lawvere, as in Lawvere theory or Category with Finite Products